tests/foreman/api/test_contentview.py

```diff
@@ -428,6 +428,48 @@ def test_positive_promote_3(self):
         self.assertEqual(len(cvv_attrs['environments']), REPEAT + 1)
         self.assertEqual(len(cvv_attrs['puppet_modules']), 1)
 
+    def test_add_normal_cv_to_composite(self):
+        """@Test: Create normal content view, publish and
+        add it to a new composite content view
+
+        @Assert: Content view can be created and assigned to
+        composite one through content view versions
+        mechanism
+
+        @Feature: ContentView
+
+        """
+        content_view = entities.ContentView(organization=self.org.id)
+        content_view.id = content_view.create_json()['id']
```

Comment:
As of [NailGun 0.7.0](https://github.com/SatelliteQE/nailgun/releases/tag/0.7.0), you can now write this instead:

    content_view = entities.ContentView(organization=self.org.id).create()

Here's a capture from my local system that further illustrates what's going on:

```python
>>> from robottelo import entities
>>> content_view = entities.ContentView(organization=1).create()
>>> type(content_view) == entities.ContentView
True
>>> content_view.id
1043
>>> content_view.get_values()
{'description': None, 'repository': [], 'composite': None, 'component': [], 'label': u'bed9f251-953f-4670-b973-150536e241a3', 'organization': <robottelo.entities.Organization object at 0x7f51ff0ffe90>, 'id': 1043, 'name': u'\u4718\U000269f5'}
>>> content_view.organization.id
1
```